pom.xml

```diff
@@ -8,78 +8,78 @@
 		<version>3.1.1</version>
 		<relativePath/> <!-- lookup parent from repository -->
 	</parent>
-	<groupId>com.nashtech</groupId>
-	<artifactId>java-competency-demo</artifactId>
+	<groupId>com.knoldus</groupId>
+	<artifactId>vehicleApp</artifactId>
 	<version>0.0.1-SNAPSHOT</version>
-	<name>java-competency-demo</name>
+	<name>vehicleApp</name>
 	<description>Demo project for Spring Boot</description>
 	<properties>
-		<java.version>19</java.version>
-		<sonar.organization>java-competency-demo</sonar.organization>
```

Comment:
Why is sonar config removed?

Reply:
Sorry it was removed by mistake, will revert back the changes in Pom.